packages/core/types/src/types/core/permissions/index.ts

```diff
@@ -1,14 +1,8 @@
-import { Subject } from '@casl/ability';
+import { engine } from '@strapi/permissions';
```

Comment:
Is the engine gotten here a type or a value?

Reply:
it was a type – changed! :)